step_07/ReadMe.md

```diff
@@ -1,9 +1,13 @@
 > You can download the code of this step [here](../src/step_07.py) or all the steps [here](https://github.com/Avaiga/taipy-getting-started-core/tree/develop/src).
 
-# Step 7: Execution modes
+# Execution modes
```

Comment:
What about the following title that looks more consistent with the previous titles?
```suggestion
# Executing jobs
```